src/stock.rs

```diff
@@ -276,6 +272,1129 @@ impl StockMessage {
     }
 }
 
+/// Builder for a stock string.
+///
+/// See [`NoMessages`] or any other stock string in this module for an example of how to use
+/// this.
+struct StockString<'a> {
+    context: &'a Context,
+}
+
+impl<'a> StockString<'a> {
+    /// Creates a new [`StockString`] builder.
+    fn new(context: &'a Context) -> Self {
+        Self { context }
+    }
+
+    /// Looks up a translation and returns a further builder.
+    ///
+    /// This will look up the translation in the [`Context`] if one is registered.  It
+    /// returns a further builder type which can be used to substitute replacement strings
+    /// or build the final message.
+    async fn id(self, id: StockMessage) -> TranslatedStockString<'a> {
```

Comment:
I wasn't sure on this either, originally i was trying to write `StockString::new(context).id(StockMessage::ReplyNoun).fallback("Reply").msg()` but that didn't work out because I couldn't reasonably get the fallback there (it's needed in `set_stock_translations()` which makes life a bit tricky.

But yeah, at this point `StockString::new(context, StockMessage::ReplyNoun).await.msg()` is probably as clear and simpler on the types.